src/main/kotlin/app/revanced/patches/youtube/layout/hidebreakingnewsshelf/resource/patch/HideBreakingNewsShelfResourcePatch.kt

```diff
@@ -0,0 +1,42 @@
+package app.revanced.patches.youtube.layout.hidebreakingnewsshelf.resource.patch
```

Comment:
This patch should be moved to the package `app.revanced.patches.youtube.layout.homepage.breakingnews.resource.patch`